compiler/src/iree/compiler/DispatchCreation/FormDispatchRegions.cpp

```diff
@@ -715,15 +715,22 @@ fuseRootsWithConsumers(MLIRContext *context, ArrayRef<Operation *> roots,
       SmallVector<OpOperand *> fusableUses =
           getFusableUses(context, currRoot, dominanceInfo,
                          /*aggressiveFusion=*/options.aggressiveFusion);
-      if (fusableUses.empty())
+      if (fusableUses.empty()) {
         continue;
+      }
+
+      // For now prune the fusable uses due to codegen failures. Ideally we
+      // should just be taking the whole set of fusable uses.
+      SmallVector<OpOperand *> prunedFusableUses;
+      prunedFusableUses.push_back(fusableUses.front());
```

Comment:
If we're going to support this, it is better to create an issue and put more context to the issue. Because future contributors may have the question when they see the comment.

Reply:
I think its a host of issues... I think future contributor would have to make the change and test where things fail :D